### PR TITLE
Aqua remotely exploitable critical vulnerabilities improvements

### PR DESCRIPTION
**Pull request by valituguran**
Update Aqua scan error messages
Fix logging
Update security vulnerability issue description
Update release status issue comment and status

### CODE REVIEW

File: src/org/ods/orchestration/util/Project.groovy [hunk 1]
```diff
@@ -1310,6 +1314,42 @@ class Project {
...
+
+        return securityVulnerabilities.collectEntries { secVul ->
+            def issue = [
+                key: secVul.key,
+                name: secVul.fields.summary,
+                assignee: secVul.fields.assignee ?
+                    [secVul.fields.assignee.displayName,
+                     secVul.fields.assignee.name,
+                     secVul.fields.assignee.emailAddress].find { it != null } : "Unassigned",
+                dueDate: '',
```
Comment (hrcornejo) [inline src/org/ods/orchestration/util/Project.groovy:1345]: Why are obtaining due date if is not used here.

File: src/org/ods/orchestration/util/GitUtil.groovy [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package org.ods.orchestration.util
```
Comment (hrcornejo) [inline src/org/ods/orchestration/util/GitUtil.groovy:1]: Needs a unit test

**Review by hrcornejo — CHANGES_REQUESTED** [review]

File: src/org/ods/orchestration/util/Project.groovy (hunk 1 above)
Comment (valituguran) [reply src/org/ods/orchestration/util/Project.groovy:1345]: Removed.

File: src/org/ods/orchestration/util/GitUtil.groovy (hunk 2 above)
Comment (valituguran) [reply src/org/ods/orchestration/util/GitUtil.groovy:1]: Added.

**Review by hrcornejo — APPROVED** [review]

**Pull request merged**